src/resources/about-food-labels/view.html

```diff
@@ -0,0 +1,498 @@
+          <a
+            href="https://www.industry.gov.au/regulations-and-standards/buying-and-selling-goods-and-services-by-weights-and-other-measures"
+            target="_blank">
+            NMI website
+          </a>
+          for more information on how to comply with the laws around weights
+          and other measures.
+        </p>
+
+        <figure>
+          <img src="https://via.placeholder.com/300x300.png/ccc/ccc" alt="" />
+        </figure>
+
+      </div>
+    </article>
+
+    <article>
+      <input
+        id="about-food-labels-4"
+        type="checkbox"
+        name="tabs"
+        tabindex="-1"
+        aria-controls="id-panel-content-1"
+        aria-expanded="false"
+        role="checkbox">
+      <h3
+        class="acc-heading">
+        <label for="about-food-labels-4">
+          <span class="title">
+            <span class="acc-number">4</span>
+            Date marks
+          </span>
+          <span class="arrow"><i></i></span>
+        </label>
+      </h3>
+      
+      <div
+        class="collapsing-section"
+        aria-hidden="true"
+        id="id-panel-content-1">
+        
+        <p>
+          Food packaged with a shelf life of 2 years or less must show a date
+          mark.
+        </p>
+        
+        <p>
+          Foods that should be eaten before a certain date for health and
+          safety reasons must be labelled with a use-by date. Otherwise, a
+          best-before date is required if the food has a shelf life of less
+          than 2 years. Although it may be safe to eat the food after the
+          best-before date, the food may have lost quality or some nutritional
+          value.
+        </p>
+
+        <p>
+          Bread with a shelf-life of less than 7 days, can use a baked-for date or baked-on date instead of a best-before date.
+        </p>
+
+        <figure>
+          <img src="https://via.placeholder.com/300x300.png/ccc/ccc" alt="" />
+          <img src="https://via.placeholder.com/300x300.png/ccc/ccc" alt="" />
+          <img src="https://via.placeholder.com/300x300.png/ccc/ccc" alt="" />
+        </figure>
+
+      </div>
+    </article>
+
+    <article>
+      <input
+        id="about-food-labels-5"
+        type="checkbox"
+        name="tabs"
+        tabindex="-1"
+        aria-controls="id-panel-content-1"
+        aria-expanded="false"
+        role="checkbox">
+      <h3
+        class="acc-heading">
+        <label for="about-food-labels-5">
+          <span class="title">
+            <span class="acc-number">5</span>
+            Lot identification
+          </span>
+          <span class="arrow"><i></i></span>
+        </label>
+      </h3>
+      
+      <div
+        class="collapsing-section"
+        aria-hidden="true"
+        id="id-panel-content-1">
+        
+        <p>
+          The lot identification means a number or other information that
+          identifies where the food was made and the batch it is was part of.
```

Comment:
"it is was" -> "it was"

Reply:
This copy has... you probably saw the other comments.

Reply:
Updated.